CHANGELOG.rst

```diff
@@ -4,6 +4,24 @@ community.aws Release Notes
 
 .. contents:: Topics
 
+v9.3.0
+======
+
+Release Summary
+---------------
+
+This release includes several improvements for the ``community.aws.aws_ssm`` connection plugin in preparation for its promotion to amazon.aws and bumps version of ansible-lint to ``25.1.0``.
+
+Minor Changes
+-------------
+ß
```

Comment:
Oops, this almost flew past the radar; I think this `ß` character needs to be removed.